### PR TITLE
Sle15 fix ansible cis remediations

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- When run in check mode ansible remediation scripts should be more isolated and need extra checks instead of assumptions

#### Rationale:

- Fix ansible SLES cis remediations in check mode

- Drop service_timesyncd_configured from CIS SLE profile

### CODE REVIEW

File: linux_os/guide/system/network/network-nftables/set_nftables_base_chain/ansible/shared.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 #Name of the table
 {{{ ansible_instantiate_variables("var_nftables_table") }}}
-#Familiy of the table 
+#Familiy of the table
```
Comment (jan-cerny) [inline linux_os/guide/system/network/network-nftables/set_nftables_base_chain/ansible/shared.yml:9]:
```suggestion
#Family of the table
```

**Review by jan-cerny — COMMENTED** [review]
In the GitHub CI in Automatus's test run on SLE15 there seems to be some problem with test scenarios of rule `all_apparmor_profiles_enforced`.


Looking at the prescripts logs, there are fail like for example:

```
incorrect_apparmor_profiles.fail.sh: line 4: aa-teardown: command not found
```

What do you think about it?

**Comment by teacup-on-rockingchair** [comment]
> In the GitHub CI in Automatus's test run on SLE15 there seems to be some problem with test scenarios of rule `all_apparmor_profiles_enforced`.
> 
> Looking at the prescripts logs, there are fail like for example:
> 
> ```
> incorrect_apparmor_profiles.fail.sh: line 4: aa-teardown: command not found
> ```
> 
> What do you think about it?
I did spend some time trying to overcome this but failed for now.

The immediate problem there is the installation of required packages, which is fixable by adding the packages requirements for the failing and similar apparmor tests.

In general the tests there would require a container with apparmor enabled, with systemd service, the proper file system and to be honest tried to fixed it by mocking the file system structure but still it fails because of some syscalls failing I believe

**Comment by teacup-on-rockingchair** [comment]
Hey @jan-cerny can you please merge this one, since it is now pending on someone from the SUSE-maintainers group but for now I am the only member of the group and cannot do much about it :)

**Pull request merged**